### PR TITLE
Bc 1447 language key fab

### PR DESCRIPTION
**Pull request by luizzaly**
# Short Description

Adjusted languages keys for FAB. Added common label "Erstellen"

## Links to Ticket and related Pull-Requests
https://ticketsystem.hpi-schul-cloud.org/browse/BC-1447

Bildschirmfoto 2022-03-17 um 15 22 28

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: src/components/atoms/vCustomFab.vue [hunk 1]
```diff
@@ -209,6 +209,10 @@ export default {
 	width: auto;
 }
 
+.v-fab-action {
+	width: max-content;
+}
+
```
Comment (MartinSchuhmacher) [inline src/components/atoms/vCustomFab.vue:215]: grafik

I added this property on my branch https://github.com/hpi-schul-cloud/nuxt-client/pull/2084 
please check if that also solves the issue you wanted to solve here, so we can be consistent without creating a new class

**Review by MartinSchuhmacher — CHANGES_REQUESTED** [review]

File: locale/de.json [hunk 2]
```diff
@@ -454,9 +455,9 @@
 	"pages.administration.students.new.error": "Schüler:in konnte nicht erfolgreich hinzugefügt werden. Möglicherweise existiert die E-Mail-Adresse bereits.",
 	"pages.administration.students.new.success": "Schüler:in erfolgreich angelegt!",
 	"pages.administration.students.new.title": "Schüler:in hinzufügen",
-	"pages.administration.teachers.fab.add": "Anlegen",
+	"pages.administration.teachers.fab.add": "Lehrkraft anlegen",
 	"pages.administration.teachers.fab.add.aria": "Lehrkraft anlegen",
-	"pages.administration.teachers.fab.import": "Importieren",
+	"pages.administration.teachers.fab.import": "Lehrkraft importieren",
 	"pages.administration.teachers.fab.import.aria": "Lehrkraft importieren",
```
[the remark this answers is not in the document]
Comment (monaweitzenberg) [reply locale/de.json]: solved!

File: locale/de.json [hunk 3]
```diff
@@ -669,9 +670,9 @@
 	"pages.rooms.groupName": "Kurse",
 	"pages.rooms.a11y.group.text": "{title}, Ordner, {itemCount} Kurse, zum Öffnen die Eingabetaste drücken",
 	"pages.rooms.headerSection.ariaLabel": "Liste der aktuellen Kurse {itemCount} Elemente",
-	"pages.rooms.fab.add": "Erstellen",
-	"pages.rooms.fab.add.aria": "Kurs erstellen",
-	"pages.rooms.fab.import": "Importieren",
+	"pages.rooms.fab.add": "Neuer Kurs ",
```
[the remark this answers is not in the document]
Comment (monaweitzenberg) [reply locale/de.json:666]: solved!

File: locale/de.json [hunk 4]
```diff
@@ -669,9 +670,9 @@
...
+	"pages.rooms.fab.add.aria": "Neuen Kurs erstellen",
+	"pages.rooms.fab.import": "Kurs importieren",
 	"pages.rooms.fab.import.aria": "Kurs importieren",
```
[the remark this answers is not in the document]
Comment (monaweitzenberg) [reply locale/de.json]: solved!

File: src/components/atoms/vCustomFab.vue (hunk 1 above)
Comment (monaweitzenberg) [reply src/components/atoms/vCustomFab.vue:212]: solved!

File: src/components/atoms/vCustomFab.vue [hunk 5]
```diff
@@ -37,7 +37,7 @@
 		</template>
 		<template v-for="(action, index) of actions">
 			<div :key="index" class="d-flex align-center justify-end">
-				<span>
+				<span class="v-fab-action">
```
[the remark this answers is not in the document]
Comment (monaweitzenberg) [reply src/components/atoms/vCustomFab.vue]: That makes sense :) removed

**Review by MartinSchuhmacher — APPROVED** [review]

**Pull request merged**